modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -1,15 +1,15 @@
 [id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
 = Configuring JSON log data for Elasticsearch
 
-When forwarding JSON logs to an Elasticsearch log store, you must create an index for each format if the JSON log entries _have different formats_.
+If your JSON logs have more than one schema, storing them in a single index might cause type conflicts and cardinality problems. To avoid that, you must configure the `ClusterLogForwarder` custom resource (CR) to group each schema into a single output definition. This way, each schema is forwarded to a separate index.
+
+If you forward to Red Hat's OpenShift Logging managed Elasticsearch, it will try and generate a new index based on your configuration.
 
 [IMPORTANT]
 ====
-You must create a separate index for each different JSON log format. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+To avoid performance issues associated with having too many indices, keep the  number of possible schemas low by standardizing to common schemas.
```

Comment:
can we say 
> ..., we recommend to keep ...

Reply:
seems there is also an additional whitespace in between "the" and "number"

Reply:
Updated. But, unfortunately, I'm not allowed to use "we" or "recommend". If we want to recommend something, we simply tell them to do it, like we've done here.  I'll soften it by saying "consider keeping..."

Reply:
Updated. Thanks.